### DISCUSSION TITLE
Issues w/v4b5:  Long Light blink at Power Up and end of Local Sequence

### DISCUSSION THREAD

**Discussion by sirbwolf** [forkineye/ESPixelStick#763] [Q&A] [answered]
On power up, and also at the end of a sequence, my lights seem to turn on and stay white for a few seconds, and then shut off.  I can replicate this on an immediate STOP or Graceful stop.

Second issue is the main interface page for the PixelStick isn't completely rendering, and I'm observing the following error on the serial console:  Status Doc is too Small.

![image](https://github.com/forkineye/ESPixelStick/assets/171988796/547d8cd7-57cd-4997-8f85-11c006d7eea4)
![image](https://github.com/forkineye/ESPixelStick/assets/171988796/06208678-5d8f-46df-88a3-777e23faee88)

**Comment by MartinMueller2003** [comment]
What version are you loading up? That status issue was fixed about 4 months ago. Are you using the latest CI build or a newer build that is on the Web installer?
Dist at https://github.com/MartinMueller2003/ESPixelStick/releases
And the url for the web flash tool tool: https://espixelstickwebflasher.from-ct.com:5000/

**Comment by sirbwolf** [comment]
I'm running the latest Beta release - 4.0-beta5 / Build Date: Dec 1 2023 - 17:13:26

https://github.com/forkineye/ESPixelStick/releases/tag/v4.0-beta5

**Comment by MartinMueller2003** [comment, marked as answer]
That explains it. Please go to a newer version. I suggest using the web installer at

https://espixelstickwebflasher.from-ct.com:5000/

**Comment by sirbwolf** [comment]
Okay, that took care of it!  Apologizes for not looking at the CI builds - didn't know really know they were out there.    Thanks!!
